tests/Functional/PluginInteractionTest.php

```
@@ -102,27 +96,24 @@ public function testCachePluginUsesIpFromFakeIpPlugin(): void
-                    ->method('getStatusCode')
-                    ->willReturn(200);
-                $response->expects(self::once())
-                    ->method('getBody')
-                    ->willReturn($stream);
-
-                return $response;
-            }
-
-            self::fail(sprintf('Unexpected http call "%s %s".', $request->getMethod(), (string) $request->getUri()));
-        });
+        $container->set('http_client', new MockHttpClient(static function (string $method, string $url, array $options): JsonMockResponse {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$options'. (undefined)

(UnusedFormalParameter)